ymer/src/test/java/com/avanza/ymer/YmerInitialLoadIntegrationTest.java

```diff
@@ -58,7 +57,7 @@ public void migratesOldDocumentOnInitialLoad() throws Exception {
 		Document spaceObjectV2 = new Document();
 		spaceObjectV2.put("_id", "id_v2");
 		spaceObjectV2.put("message", "Msg_V2");
-		spaceObjectV2.put(MirroredObject.DOCUMENT_FORMAT_VERSION_PROPERTY, 2);
+		spaceObjectV2.put(MirroredObject.DOCUMENT_FORMAT_VERSION_PROPERTY, 3);
```

Comment:
Just for my understanding, why did the version property needed to be increased here? Was the test incorrect or is there any changes that affected this?

Reply:
My understanding of the original test was to ensure that this object does not get patched by the supplied patchers. Its version was 2 originally, and it is asserted on line 84 that teh version is not changed after running all patchers. In this commit, we add another patch object, thus increasing the resulting document version to 3 for all objects. To keep the original intention of "this object should not be patched", the version is incremented here.